routemaster-client.gemspec

```diff
@@ -19,5 +19,5 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency     'typhoeus', '~> 1.1'
   spec.add_runtime_dependency     'faraday', '>= 0.9.0'
   spec.add_runtime_dependency     'wisper', '~> 1.6.1'
-  spec.add_runtime_dependency     'oj', '~> 2.17'
```

Comment:
Seeing as users might still use either Oj 2 or Oj 3, I think we need to either:
- test against both versions using `appraisal`
- pin to Oj >= 3?